### ISSUE TITLE
test

### ISSUE THREAD

**Issue by seblemaguer** [marytts/marytts-http-server#7]
We need a set of test to analyze a basic process with travis

**Comment by attacomsian** [comment]
Tests are added in #8.

**Comment by seblemaguer** [comment]
Produce the following error:

```
marytts.http.WebLayerTest > initializationError FAILED
    java.lang.StackOverflowError

1 test completed, 1 failed
```

**Comment by seblemaguer** [comment]
solved by 5c96bcf
